sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
> I don't see why fixing a deprecation warning in Sufia's test suite should force every Sufia adopter to update their devise dependency to ~> 4.2.

👏  since the application works fine with older versions of devise, forcing people to upgrade to save ourselves a couple lines of (test-only) code doesn't seem worth the inconvenience it can cause downstream. Now, if it were a functional change to production code, I might be more wary of trying to maintain compatibility, but this is entirely sufia-developer facing and doesn't strike me as an undue burden.